index.html

```diff
@@ -58,16 +138,46 @@ <h1 class="logo">
     <section class="About-section">
       <div class="about">About</div>
```

Comment:
- [ ] Kindly change the `<div class="about">` to one of the `hx(h2-h6)` tags. Using `hx` tags for name titles or subtitles makes your HTML code semantic. 👍